pkg/services/pluginsintegration/angularpatternsstore/store.go

```diff
@@ -1,18 +1,12 @@
 package angularpatternsstore
 
 import (
-	"context"
-	"encoding/json"
-	"fmt"
-	"time"
-
 	"github.com/grafana/grafana/pkg/infra/kvstore"
+	"github.com/grafana/grafana/pkg/services/pluginsintegration/cachekvstore"
 )
 
 type Service interface {
-	Get(ctx context.Context) (string, bool, error)
-	Set(ctx context.Context, patterns any) error
-	GetLastUpdated(ctx context.Context) (time.Time, error)
+	cachekvstore.SingleKeyStore
```

Comment:
can you make this a NamespacedStore? So you don't need two different implementations for the same store.